RELEASING.md

```diff
@@ -8,45 +8,47 @@ The Google Cloud Ruby Client project uses [semantic versioning](http://semver.or
 
 After all [pull requests](https://github.com/GoogleCloudPlatform/google-cloud-ruby/pulls) for a release have been merged and all [Circle CI builds](https://circleci.com/gh/GoogleCloudPlatform/google-cloud-ruby) are green, you may create a release as follows:
 
-1. Build the gem locally.
+1. If you haven't already, switch to the master branch, ensure that you have no changes, and pull from origin.
 
     ```sh
-    $ rake build
+    $ git checkout master
+    $ git status
+    $ git pull <remote> master --rebase
     ```
 
-1. Install the gem locally.
+1. Build the gem locally. (Depending on your environment, you may need to `bundle exec` to rake commands; this will be shown.)
 
     ```sh
-    $ rake install
+    $ cd <gem>
+    $ bundle exec rake build
     ```
 
-1. Using IRB (not `rake console`!), manually test the gem that you installed in the previous step.
-
-1. Review the changes since the last release.
+1. Install the gem locally.
 
     ```sh
-    $ bundle exec rake changes[<gem>]
+    $ bundle exec rake install
     ```
 
-1. Review the commits in the changes output, making notes of significant changes. (For examples of what a significant change is, browse the changes in the gem's `CHANGELOG.md`
+1. Using IRB (not `rake console`!), manually test the gem that you installed in the previous step.
```

Comment:
For some reason, with a few of the gems I had to `bundle exec irb` or else I would get a `Gem::MissingSpecError` when I tried to `require` the gem. IIRC, this only happened for `stackdriver` and `debugger`.

Reply:
Interesting! I'll take a look at those gems to see what might be different.

Reply:
Debugger is not included in google-cloud because it requires a more recent version of Ruby than the other gems.

Reply:
Actually, this might be something different. Investigating.

Reply:
Maybe we can recommend `gem install` in a clean gemset for this step, then?

Reply:
In general, I would like to avoid requiring gemset functionality to do releases because the mechanics are different depending on what Ruby version manager folks are using. But, it might be worth noting that Bundler's `install` task may not always work as expected, and to fall back onto using `rake build`, `gem install`, and an empty gemset to verify this if you run into issues with `rake install`.

Chris, what do you think?

Reply:
I'll try it too, but success apparently won't be proof of anything. I'm open to adding a version of:

> Bundler's install task may not always work as expected, and to fall back onto using rake build, gem install, and an empty gemset to verify this if you run into issues with rake install.

Reply:
@quartzmo If you add that then I'll approve.

Reply:
@blowmage It has been added. See latest.

Reply:
```
-1. Install the gem locally.
+1. Build the gem locally. (Depending on your environment, you may need to `bundle exec` to rake commands; this will be shown.)
```

Reply:
```
-1. Install the gem locally.
 +1. Install the gem locally. (The `rake install` task shown below may not always work as expected. Fall back to running `gem install` in an empty gemset if needed.)
```

Reply:
I see it now. Thanks!